client/src/portal/mod.rs

```diff
@@ -609,4 +659,40 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn change_secret() -> Result<(), Error> {
+        let path = PathBuf::from("../../tests/test_rekeying.keyring");
+
+        let mut keyring = Keyring::load(&path, strong_key()).await?;
+        let attributes = HashMap::from([("attr", "value")]);
+        keyring
+            .create_item("test", &attributes, "password", false)
+            .await?;
+        let item_before = keyring.lookup_item(&attributes).await?;
+
+        let new_secret = Secret::from(b"password".to_vec());
+        keyring.change_secret(new_secret).await?;
+
+        let new_secret = Secret::from(b"password".to_vec());
+        let keyring = Keyring::load(&path, new_secret).await?;
+        let item_now = keyring.lookup_item(&attributes).await?;
+
+        assert_eq!(
+            item_before.clone().unwrap().label(),
```

Comment:
All the clones and unwrap here are wrong, unwrap when retrieving the item. But doesn't create_item return the item anyways?

Reply:
yes. done, thanks!